open-source-services.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Minimum Open Source requirements for Services
+description: The minimum requirements for Open Source when working on services, that then allows further progression.
+---
+
+# Minimum Open Source Requirements for Services
+
+Here we document the minimum requirements for use of Open Source in services.
+When a service meets this minimum requirements, it is then possible for contributors to further progress the service toward the complete Operate First methodology.
+This represents what we expect from any _service_ claiming to be Open Source.
+
+## Requirements
+
+These requirements do not represent best practices, but are the bare minimum for Open Source development of services.
+Most Open Source models (beyond “throw it over the wall” behavior) are covered by these requirements in one way or another:
+
+ 1. **All code and assets necessary to operate the service are shared under an open source license, and publicly accessible.**
+ 1. **A public contributor can use the same workflow as a typical team member to make a change to the service.**
```

Comment:
```suggestion
 2. **A public contributor can use the same workflow as a typical team member to make a change to the service.**
```
I know MarkDown does the magic, but we refer to these requirements by number in this doc, so let's make the numbering just as obvious in the MD source.